app/controllers/reports/regions_controller.rb

```diff
@@ -93,6 +105,19 @@ def whatsapp_graphics
 
   private
 
+  def accessible_district?(district)
```

Comment:
We need a permissions cache to determine if we should show just the name of a district or block in the index view, or if we should show a link to the report of that particular region.